src/ucp/api/ucp.h

```diff
@@ -2100,6 +2127,30 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys
+ *
+ * This routine compares two remote keys.
+ *
+ * It sets the @a result argument to < 0 if rkey1 is lower than rkey2, 0 if they
+ * are equal or > 0 if rkey1 is greater than rkey2. It can be used for sorting.
```

Comment:
`It can be used for sorting` -> `e.g. the result value can be used for sorting keys.`

Reply:
fixed